vizro-core/docs/index.md

```diff
@@ -54,10 +36,12 @@ Vizro is a toolkit for creating modular data visualization applications.
 
     [:octicons-arrow-right-24: Contribute to our docs](pages/explanation/documentation-style-guide.md)
 
-- :fontawesome-solid-chart-column:{ .lg .middle } __Vizro-AI__
+- :fontawesome-solid-chart-column:{ .lg .middle } __Gen AI assistance__
```

Comment:
```suggestion
- :fontawesome-solid-chart-column:{ .lg .middle } __Vizro with Gen AI__
```

Or similar? "assistance" just sounds a bit weird to me, but if it sounds natural to you then fine by me.